webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
+
+module.exports = {
+  entry: "./index.js",
+  output: {
+    path: path.resolve(__dirname, "dist"),
+    filename: "app.js",
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ["style-loader", "css-loader"],
+      },
+      {
+        test: /\.ttf$/,
+        use: ["file-loader"],
+      },
+    ],
+  },
+  plugins: [
+    new MonacoEditorPlugin({
+      // https://github.com/Microsoft/monaco-editor-webpack-plugin#options
+      // Include a subset of languages support
+      // Some language extensions like typescript are so huge that may impact build performance
+      // e.g. Build full languages support with webpack 4.0 takes over 80 seconds
+      // Languages are loaded on demand at runtime
+      languages: ["javascript", "css", "html", "typescript"],
```

Comment:
Could you also add json in here?

Reply:
Sorry, I accidentally deleted it but it somehow still worked. I will add "json" in next commit and delete the other unused languages.

Reply:
I think it is safe to assume that there will be js, ts and json edited in wade for foreseeable future. Maybe plantuml files as well later on.